includes/class-sensei-course-progression-settings.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * File containing the Sensei_Course_Progression_Settings class.
+ *
+ * @package sensei-lms
+ * @since 4.0.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Adds the course progression settings to the course.
+ *
+ * @since 4.0.0
+ */
+class Sensei_Course_Progression_Settings {
```

Comment:
What do you think about putting this file inside its own `course-video` folder to make maintenance a bit easier? Should we also have `Video` in the class name to make it clear that it only pertains to videos? Something like `Sensei_Course_Video_Settings` might work.